test/models/messages/ArrivalNotificationSpec.scala

```diff
@@ -432,5 +431,97 @@ class ArrivalNotificationSpec extends FreeSpec with MustMatchers
           Json.toJson(notification: ArrivalNotification) mustEqual json
       }
     }
+
+    "must convert to valid xml" in {
```

Comment:
If we move the xsd into a service we could create another spec and move all validation checks into this